content/riak/kv/2.2.0/using/performance/open-files-limit.md

```diff
@@ -15,326 +15,278 @@ aliases:
-init scripts.
-
-On CentOS/RedHat systems, make sure to set a proper limit for the user
-you’re usually logging in with to do any kind of work on the machine,
-including managing Riak. On CentOS, `sudo` properly inherits the values
-from the executing user.
-
-[Reference](http://www.cyberciti.biz/faq/linux-increase-the-maximum-number-of-open-files/)
+This file is automatically sourced from the initialization script, and the Riak KV process will inherit this setting. Since initialization scripts are always run as the root user, there’s no need to set limits in /etc/security/limits.conf.
 
 ### Enable PAM-Based Limits for Debian & Ubuntu
 
-It can be helpful to enable PAM user limits so that non-root users, such
-as the `riak` user, may specify a higher value for maximum open files.
-For example, follow these steps to enable PAM user limits and set the
-soft and hard values *for all users of the system* to allow for up to
-*65536* open files.
+You can enable PAM-based user limits so that non-root users, such as the `riak` user, may specify a higher value for maximum open files.
+
+For example, follow these steps to enable PAM-based limits for all users to allow a maximum of 200000 open files.
 
-1\.  Edit `/etc/pam.d/common-session` and append the following line:
+1\.  Edit /etc/pam.d/common-session and addd the following line:
 
-```config
+```/etc/pam.d/common-session
 session    required   pam_limits.so
 ```
 
-2\. Save and close the file.  If `/etc/pam.d/common-session-noninteractive` exists, append the same line as above.
+2\. Save and close the file. If /etc/pam.d/common-session-noninteractive exists, append the same line as above.
 
-3\. Edit `/etc/security/limits.conf` and append the following lines to the
-file:
+3\. Edit /etc/security/limits.conf and append the following lines to the file:
 
-```bash
-soft     nofile          65536
-hard     nofile          65536
+```/etc/security/limits.conf
+* soft nofile 65536
+* hard nofile 200000
 ```
 
 4\. Save and close the file.
 
-5\. (**Optional**) If you will be accessing the Riak nodes via secure shell
-(ssh), you should also edit `/etc/ssh/sshd_config` and uncomment the
-following line:
+5\. (**Optional**) If you will be accessing the Riak KV nodes via secure shell (SSH), you should also edit /etc/ssh/sshd_config and uncomment the following line:
 
-```config
+```/etc/ssh/sshd_config
 #UseLogin no
 ```
 
-and set its value to `yes` as shown here:
+And set its value to `yes` as shown here:
 
-```config
+```/etc/ssh/sshd_config
 UseLogin yes
 ```
 
-6\. Restart the machine so that the limits to take effect and verify that
-the new limits are set with the following command:
+6\. Restart the machine so the limits take effect and verify that the new limits are set with the following command:
 
 ```bash
 ulimit -a
 ```
 
+{{% note %}}
+In the above examples, the open files limit is raised for all users of the system. The limit can be specified for the riak user only by substituting the
```

Comment:
Above, it was written `riak` user. I'm fine with that, but it should be consistent.